aspnetcore/grpc/browser.md

```diff
@@ -13,15 +13,27 @@ By [James Newton-King](https://twitter.com/jamesnk)
 
 It's not possible to directly call a gRPC service from a browser. gRPC uses HTTP/2 features, and no browser provides the level of control required over web requests to support a gRPC client.
 
-gRPC on ASP.NET Core offers two browser-compatible solutions:
+gRPC on ASP.NET Core offers two browser-compatible solutions: gRPC-Web and gRPC JSON transcoding
 
-* **gRPC-Web** allows browser apps to call gRPC services with the gRPC-Web client and Protobuf. gRPC-Web requires the browser app to generate a gRPC client. gRPC-Web allows browser apps to benefit from the high-performance and low network usage of gRPC.
+## gRPC-Web
 
-  .NET has built-in support for gRPC-Web. For more information, see <xref:grpc/grpcweb>.
+gRPC-Web allows browser apps to call gRPC services with the gRPC-Web client and Protobuf.
 
-* **gRPC JSON transcoding** allows browser apps to call gRPC services as if they were RESTful APIs with JSON. The browser app doesn't need to generate a gRPC client or know anything about gRPC. RESTful APIs can be automatically created from gRPC services by annotating the `.proto` file with HTTP metadata. Transcoding allows an app to support both gRPC and JSON web APIs without duplicating the effort of building separate services for both.
+* Very similar to normal gRPC but it has a slightly different wire-protocol which makes it compatible with HTTP/1.1 and browsers.
```

Comment:
*NIT*: Words like "very" don't add *very* 😄 much value.

There are two ways to structure this ...

```suggestion
* Similar to normal gRPC, but it has a slightly different wire-protocol, which makes it compatible with HTTP/1.1 and browsers.
```

```suggestion
* Similar to normal gRPC but with a slightly different wire-protocol, which makes it compatible with HTTP/1.1 and browsers.
```